src/components/ReactTags.js

```diff
@@ -21,6 +21,14 @@ import {
 
 import '../styles/react-tags.scss';
 
+const updateClass  = (classNames) =>
```

Comment:
can you add a TODO in line 104 to remove classNames from state and change updateClass to instance function.